src/ui/hooks/useOnlineStatusEffect.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from "react";
+import { useAppSelector } from "../../store/hooks";
+import { getIsOnline } from "../../store/reducers/stateCache";
+
+const useOnlineStatusEffect = (
```

Comment:
Looks like this is used to make sure the page is loaded after we come back online - great work